_dashboards/discover/exploring-query-enhancements.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: Exploring query enhancements
+parent: Analyzing data
+nav_order: 20
+---
+
+# Exploring query enhancements
+Introduced 2.17
+{: .label .label-purple }
+
+Starting with OpenSearch 2.17, query enhancements have been made. These enhancements are experimental and may be subject to change or instability. Enhancements include the following: 
+
+- Query languages PPL and SQL, with **Query Assist** for PPL
+- Multiline query editor for PPL and SQL and autocomplete for PPL and DQL
+- Data selector with **Data Explorer** that supports index patterns, indexes, and Amazon S3 connections and data configuration that supports selecting the appropriate query language based on the data type
+- Link sharing through URLs without needing write permission to create an index pattern
+
+All existing functionality remains in OpenSearch 2.17, and the new features are designed to improve your data exploration experience. The **Enable query enhancements** is currently not on a minimum distribution. <SME: Do we need to add text about plugins?>. You can leave your feedback at <insert-link> to help the OpenSearch Project improve.
```

Comment:
- In a distribution of just `OpenSearch` and `OpenSearch Dashboards`, query enhancements will not provide access to PPL or SQL. This requires the OpenSearch SQL plugin. I think we can make it known that query enhancements does work with just OpenSearch, but will require the SQL plugin and others to make SQL, PPL, and external datasource queries work.
- Other than making a note for getting SQL and PPL to work, looks good to me. May need some more input from @kavilla as well.

Reply:
Revised the Note text to add this information.